File: language-samples/hello.cs
/**
 * C# sample for GitHub language detection
 * BarbrickDesign - Complete Language Portfolio
 */

using System;
using System.Collections.Generic;
using System.Linq;

namespace BarbrickDesign
{
    class Greeter
    {
        private string message;
        
        public Greeter(string message)
        {
            this.message = message;
        }
        
        public void Greet()
        {
            Console.WriteLine(message);
        }
    }
    
    class Program
    {
        static void Main(string[] args)
        {
            var greeter = new Greeter("Hello from C#!");
            greeter.Greet();
            
            // List
            var languages = new List<string> { "C#", "Java", "Python" };
            
            // Dictionary
            var info = new Dictionary<string, string>
            {
                ["language"] = "C#",
                ["paradigm"] = "Multi-paradigm",
                ["typing"] = "Static"
            };
            
            // LINQ
            var upper = languages.Select(l => l.ToUpper());
            foreach (var lang in upper)
            {
                Console.WriteLine(lang);
            }
        }
    }
}

File: language-samples/Hello.java
/**
 * Java sample for GitHub language detection
 * BarbrickDesign - Complete Language Portfolio
 */

import java.util.*;

public class Hello {
    private String message;
    
    public Hello(String message) {
        this.message = message;
    }
    
    public void greet() {
        System.out.println(this.message);
    }
    
    public static void main(String[] args) {
        Hello greeter = new Hello("Hello from Java!");
        greeter.greet();
        
        // Collections
        List<String> languages = new ArrayList<>();
        languages.add("Java");
        languages.add("Python");
        languages.add("JavaScript");
        
        // HashMap
        Map<String, String> info = new HashMap<>();
        info.put("language", "Java");
        info.put("paradigm", "Object-oriented");
        info.put("typing", "Static");
        
        // Stream API
        languages.stream()
                 .map(String::toUpperCase)
                 .forEach(System.out::println);
    }
}
